File: SIN BDD/RESTFULL_JAVA_CONUNI_GR06/08. SERVIDOR/servidor_restful_java_conuni_gr06/src/java/ec/edu/monster/servicio/ServicioMasa.java
package ec.edu.monster.servicio;

public class ServicioMasa {

    public double kilogramosALibras(double kilogramos) {
        return kilogramos * 2.20462;
    }

    public double gramosAOnzas(double gramos) {
        return gramos * 0.035274;
    }

    public double toneladasAKilogramos(double toneladas) {
        return toneladas * 1000.0;
    }

    public double librasAOnzas(double libras) {
        return libras * 16.0;
    }

    public double miligramosAGramos(double miligramos) {
        return miligramos / 1000.0;
    }
}

File: SIN BDD/SOAP_DOTNET_CONUNI_GR06/08. SERVIDOR/servidor_soap_dotnet_conuni_gr06/src/Servicio/ServicioMasa.cs
namespace Ec.Edu.Monster.Servicio;

public class ServicioMasa
{
    public double KilogramosALibras(double kilogramos) => kilogramos * 2.20462;

    public double GramosAOnzas(double gramos) => gramos / 28.3495;

    public double ToneladasAKilogramos(double toneladas) => toneladas * 1000;

    public double LibrasAOnzas(double libras) => libras * 16;

    public double MiligramosAGramos(double miligramos) => miligramos / 1000;
}
